CHANGELOG.md

```diff
@@ -1 +1,3 @@
 - Fixes bug causing Hosting emulator to serve invalid /\_\_/\* files.
+- Fixes bug in Firestore emulator for evaluating rules in query listening.
```

Comment:
Nit: if this was reported on GitHub add the issue number to the changelog?

Reply:
Oops, too late.

Reply:
Oops too soon, I'll link back on GitHub.

Reply:
https://github.com/firebase/firebase-tools/pull/2260